package.json

```diff
@@ -1,85 +1,130 @@
-  "author": "epicmax <hello@epicmax.co>",
-  "main": "src/lib.js",
-  "types": "src/lib.d.ts",
-  "private": false,
+  "version": "2.0.0",
   "license": "MIT",
+  "private": false,
+  "description": "Easy to use css spinners collection with vue.js integration.",
+  "author": "epicmax <hello@epicmax.co>",
+  "homepage": "https://epic-spinners.epicmax.co",
   "url": "https://github.com/epicmaxco/epic-spinners",
   "repository": {
     "type": "git",
     "url": "https://github.com/epicmaxco/epic-spinners.git"
   },
+  "bugs": {
+    "url": "https://github.com/epicmaxco/epic-spinners/issues"
+  },
+  "keywords": [
+    "spinner",
+    "vuejs",
+    "vue",
+    "vue3"
+  ],
+  "files": [
+    "dist"
+  ],
+  "type": "module",
+  "main": "./dist/cjs/index.js",
+  "module": "./dist/es/index.js",
+  "jsdelivr": "dist/iife/index.js",
+  "unpkg": "dist/iife/index.js",
+  "types": "./dist/types/index.d.ts",
+  "exports": {
+    ".": {
+      "import": {
+        "node": "./dist/esm-node/index.mjs",
+        "default": "./dist/es/index.js"
+      },
+      "require": "./dist/cjs/index.js",
+      "default": "./dist/es/index.js"
+    },
+    "./css": "./dist/style.css",
+    "./types": "./dist/types/index.d.ts",
+    "./package.json": "./package.json"
+  },
+  "sideEffects": [
+    "*.css"
+  ],
   "scripts": {
-    "prepublishOnly": "npm run build",
-    "dev": "npm-run-all --parallel dev:lib dev:docs",
-    "dev:lib": "webpack --config build/webpack.lib.conf.js --watch --progress --hide-modules",
-    "dev:docs": "node build/dev-server.js",
-    "build": "npm run build:lib && npm run build:docs",
-    "build:lib": "node build/build-lib.js",
-    "build:docs": "node build/build.js",
-    "lint": "eslint --ext .js,.vue src"
+    "prepare": "husky install",
+    "dev": "vite",
+    "build": "tsx build/build.ts",
+    "preview": "vite preview",
+    "types:check": "vue-tsc --noEmit",
+    "lint:check": "eslint . --ext .ts,.cjs,.js,.vue,.json",
+    "lint": "npm run lint:check -- --fix",
+    "lint:style:check": "stylelint ./src/**/*.{vue,css,scss,html,md}",
+    "lint:style": "npm run lint:style:check --- --fix",
+    "format:check": "stylelint-config-prettier-check && stylelint-config-prettier-scss-check && prettier . --check",
+    "format": "prettier . --write --ignore-unknown",
+    "code:check": "npm run format:check && npm run lint:style:check && npm run lint:check && npm run types:check",
+    "code:fix": "npm run format && npm run lint:style && npm run lint"
+  },
+  "lint-staged": {
+    "*.{cjs,js,ts}": [
+      "prettier --write",
+      "eslint --fix"
+    ],
+    "*.vue": [
+      "prettier --write",
+      "stylelint --fix",
+      "eslint --fix"
+    ],
+    "*.{css,scss,html,md}": [
+      "prettier --write",
+      "stylelint --fix"
+    ],
+    "*.json": [
+      "prettier --write",
+      "eslint --fix"
+    ]
   },
   "peerDependencies": {
-    "vue": "^2.5.2"
+    "vue": "^3.2.0"
```

Comment:
Must also be in dev deps. For me, vue haven't installed for some reasone in `yarn v1.22.19`.